.github/workflows/unit_tests_with_released_puppet_gem.yaml

```diff
@@ -17,8 +17,10 @@ jobs:
         include:
           - puppet_version: '7'
             ruby: '2.7'
+            puppet_forge_token: ''
           - puppet_version: '8'
             ruby: '3.1'
+            puppet_forge_token: 'YES'
```

Comment:
Haha, nice! This reinforces my thoughts that `Gemfile.erb` should allow the source for puppetcore gems to be specified directly instead of inferring it from whether a token is present. But that's for another day.